xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1125,7 +1126,9 @@ bool CGUIWindowVideoBase::OnPlayMedia(int iItem, const std::string &player)
   }
   CLog::Log(LOGDEBUG, "%s %s", __FUNCTION__, CURL::GetRedacted(item.GetPath()).c_str());
 
-  if (StringUtils::StartsWith(item.GetPath(), "pvr://recordings/active/"))
+
+  CPVRRecordingsPath path(item.GetPath());
```

Comment:
this causes issues (identifies other files as recordings). don't touch any item that does not start with pvr://
this old code was better in this case